aggie-coding-club/Rev-Registration
# McAllen Separation

## Description
Makes McAllen sections distinct by showing an icon next to them and creating a filter for McAllen sections. Note that this merges into the filters branch since it's needed to make the McAllen filter.

## How to test
Make migrations and scrape to make McAllen sections show up, then look at a course with them (such as POLS 207 for Fall 2021) to make sure they're being handled correctly.

## Screenshots
|Before|After|
|--|--|
| ![image](https://user-images.githubusercontent.com/28655462/143793460-726f7f34-134b-491b-83fe-a756527dc6c3.png) | ![image](https://user-images.githubusercontent.com/28655462/143793491-82c074e2-a231-4c2c-9693-8cdd7454dd38.png) |

## Related tasks
Closes #565.
